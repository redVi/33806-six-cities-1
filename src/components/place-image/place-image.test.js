import React from 'react';
import renderer from 'react-test-renderer';
import PlaceImage from "@/components/place-image/place-image";

describe(`PlaceImage`, () => {
  it(`renders correctly with props`, () => {
    const tree = renderer.create(<PlaceImage img={`apartment.jpg`}/>).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it(`renders correctly without props`, () => {
    const tree = renderer.create(<PlaceImage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
