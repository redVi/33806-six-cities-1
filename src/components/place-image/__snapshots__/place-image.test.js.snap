// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlaceImage renders correctly with props 1`] = `
<div
  className="cities__image-wrapper place-card__image-wrapper"
>
  <a
    href="#"
    onClick={[Function]}
  >
    <img
      alt="Place image"
      className="place-card__image"
      height="200"
      src="img/apartment.jpg"
      width="260"
    />
  </a>
</div>
`;

exports[`PlaceImage renders correctly without props 1`] = `
<div
  className="cities__image-wrapper place-card__image-wrapper"
>
  <a
    href="#"
    onClick={[Function]}
  >
    <img
      alt="Place image"
      className="place-card__image"
      height="200"
      src="img/room.jpg"
      width="260"
    />
  </a>
</div>
`;
